scripts/llnl_scripts/move_unused_tpl_dirs_to_graveyard.py

```diff
@@ -40,23 +40,25 @@ def main():
 
     summary_file ="tpl_dirs_summary.json"
     if not os.path.exists(summary_file):
-        print("Error: Summary file from find_unused_tpl_dirs.py did not exist: {0}", summary_file)
+        print(f"Error: Summary file from find_unused_tpl_dirs.py did not exist: {summary_file}")
         return False
 
     r = json.load(open(summary_file))
 
-    print ""
-    print "[# of referenced %d ]" % len(r["referenced"])
-    print "[# of found %d ]" % len(r["found"])
-    print "[# of active %d ]" % len(r["active"])
-    print "[# of unused %d ]" % len(r["unused"])
+    print("")
+    print("[# of referenced {}]".format(len(r["referenced"])))
+    print("[# of found {}]".format(len(r["found"])))
+    print("[# of active {}]".format(len(r["active"])))
+    print("[# of unused {}]".format(len(r["unused"])))
 
     success = True
     for d in r["unused"]:
-        cmd = "mv %s %s" % (d, graveyard_path(d)) 
-        res = sexe(cmd)
-        if res != 0:
-            success = False
+        if os.path.exists(d):
+            cmd = "mv %s %s" % (d, graveyard_path(d)) 
```

Comment:
what we definitely need is three different ways of formatting strings in ~20 lines...

Reply:
I'll fix it in a follow-up PR